py/pyis-python/tests/test_gpt2_tokenizer/test_gpt2_tokenizer.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+import transformers
+import unittest
+from pyis.python import ops
+import os
+
+class TestBertTokenizer(unittest.TestCase):
+    
+    def _run_basic_test(self, query):
+        self.assertEqual(self.tokenizer.tokenize(query), self.standard.tokenize(query))
+    
+    def test_tokenize(self):
```

Comment:
Try some unicode cases, such as "😃 Smileys & People ✨Sparkles](https://emojipedia.org/sparkles/)".
Besides, encode API(`encode`, `encode2`, `encode_plus`, `encode_plus2`) should also be covered. You can add it in `_run_basic_test`